Source/WebCore/Modules/mediastream/libwebrtc/LibWebRTCDataChannelHandler.cpp

```diff
@@ -188,11 +186,11 @@ void LibWebRTCDataChannelHandler::OnMessage(const webrtc::DataBuffer& buffer)
 {
     Locker locker { m_clientLock };
     if (!m_hasClient) {
-        auto* data = buffer.data.data<uint8_t>();
+        auto data = unsafeMakeSpan(buffer.data.data<uint8_t>(), buffer.size());
```

Comment:
Would be nice to add a `span(const webrtc::DataBuffer&)` so that most code doesn't need to say unsafe.